app/controllers/comments_controller.rb

```diff
@@ -1,6 +1,16 @@
 class CommentsController < ApplicationController
   load_and_authorize_resource
 
+  def index
+    @post = Post.find(params[:post_id].to_i)
+    @posts_comments = @post.comments
```

Comment:
- [ ] You did a great job so far ,i will recomend you to  add /api to URL or api/v1 with version number in order to identify as an API based URL, 
- [ ] kindly generate a controler for api in a professional way there should be a Api/v1 folder in your controller folder This is a common convention used to distinguish between different versions of an API. As an API evolves over time, changes may be made to the endpoints, response formats, or request parameters.  see example below 

![Screenshot 2023-11-18 221640](https://github.com/Graciano1997/technology_blog_app/assets/92458236/ee2df7f9-6919-4848-977e-2c6b88873fba)

Reply:
well resolved! ✔️